packages/azure-http-specs/specs/azure/resource-manager/operation-templates/lro.tsp

```diff
@@ -0,0 +1,262 @@
+    final-state-via: Azure-AsyncOperation
+    
+    Expected verb: PUT
+    Expected path: /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.OperationTemplates/lroResources/lro
+    Expected query parameter: api-version=2023-12-01-preview
+    Expected request body:
+    ```json
+    {
+      "location": "eastus",
+      "properties": {
+        "description": "valid"
+      }
+    }
+    ```
+    Expected status code: 201
+    Expected response header: Azure-AsyncOperation={endpoint}/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/lro_create/aao
+    Expected response body:
+    ```json
+    {
+      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.OperationTemplates/lroResources/lro",
+      "name": "lro",
+      "type": "Azure.ResourceManager.Resources/lroResources",
+      "location": "eastus",
+      "properties": {
+        "description": "valid",
+        "provisioningState": "InProgress"
+      },
+      "systemData": {
+        "createdBy": "AzureSDK",
+        "createdByType": "User",
+        "createdAt": <any date>,
+        "lastModifiedBy": "AzureSDK",
+        "lastModifiedAt": <any date>,
+        "lastModifiedByType": "User",
+      }
+    }
+    ```
+    
+    Expected verb: GET
+    Expected URL: {endpoint}/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/lro_create/aao
+    
+    Expected status code: 200
+    Expected response body:
+    ```json
+    {
+      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/lro_create/aao",
+      "name": "aao",
+      "startTime": "2024-11-08T01:41:53.5508583+00:00",
+      "status" : "InProgress"
+    }
+    ```
+    
+    Expected verb: GET
+    Expected URL: {endpoint}/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/lro_create/aao
+    
+    Expected status code: 200
+    Expected response body:
+    ```json
+    {
+      "id": "/subscriptions/00000000-0000-0000-0000-000000000000/locations/eastus/lro_create/aao",
+      "name": "aao",
+      "status" : "Succeeded",
+      "startTime": "2024-11-08T01:41:53.5508583+00:00",
+      "endTime": "2024-11-08T01:42:41.5354192+00:00",
+      "properties": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test-rg/providers/Azure.ResourceManager.OperationTemplates/lroResources/lro",
+        "name": "lro",
+        "type": "Azure.ResourceManager.Resources/lroResources",
+        "location": "eastus",
+        "properties": {
+          "description": "valid",
+          "provisioningState": "Succeeded"
+        },
+        "systemData": {
+          "createdBy": "AzureSDK",
+          "createdByType": "User",
+          "createdAt": <any date>,
+          "lastModifiedBy": "AzureSDK",
+          "lastModifiedAt": <any date>,
+          "lastModifiedByType": "User",
+        }
+      }
+    }
```

Comment:
Yes, it's AAO response. I added property `properties` mentioned in the [guideline](https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/async-api-reference.md#azure-asyncoperation-resource-format), though I haven't seeing that in any real service.
![image](https://github.com/user-attachments/assets/54ae4238-2cf8-4380-a145-a2cf36559b6d)

OK, since final GET in mgmt is required, I'll remove this property.

Reply:
removed